src/compose.ts

```diff
@@ -3,7 +3,22 @@ import { DOM } from 'aurelia-pal';
 import { TaskQueue } from 'aurelia-task-queue';
 import { bindable, CompositionContext, CompositionEngine, customElement, noView, View, ViewResources, ViewSlot } from 'aurelia-templating';
 
-
+/**
+ * Available activation strategies for the view and view-model bound to compose
```

Comment:
```suggestion
 * Available activation strategies for the view and view-model bound to <compose/> element
```

Reply:
the suggestion is to highlight the usage of this, to avoid confusion between this and router `ActivationStrategy` when using auto import

Reply:
Done